### ISSUE TITLE
Doesn't work with Dart 2.16.2

### ISSUE THREAD

**Issue by aalex** [cirruslabs/docker-images-flutter#203]
Hi!
Dart 2.16.2 has been released on March 28th, 2022.

To build my Docker image for a Flutter web project, I needed to downgrade our version of Dart in our pubspec.yaml:

```
environment:
  sdk: ">=2.16.1 <3.0.0"
dependencies:
  flutter:
    sdk: flutter
```

By default, when creating a new project, it was `2.16.2`, but then building the Docker image off you base image would not work.

**Comment by sbatezat** [comment]
Hi, 

This is probably because you are using the 10.2.3 image which works with the 2.10.3 flutters SDK but.. You have created a project with the 2.10.4 SDK.

You should never use a different SDK between your local environment and your CI's one.

 Your "issue" (it's not an issue from my point of view) will be solved as soon as this PR will be merged https://github.com/cirruslabs/docker-images-flutter/pull/202

**Comment by aalex** [comment]
Hi!
Thank you for your kind answer. I can't wait for this PR to be merged!

I think that it is an issue, while we are waiting for new versions of the image to be released. I has prevented me and my team from using the latest Flutter version in our new project while using your great image for CI. If there is a way that we can help to make it easier to support the latest versions of Flutter when there is a new release, I'd be happy to contribute in some way. Perhaps we simply need to manually open a PR like #202? Or is there a way to automate this?

Thanks again.

**Comment by bartekpacia** [comment]
I think this issue can be closed, Dart 2.16.2 was released years ago.

cc @fkorotkov
